src/main/java/com/yahoo/sketches/theta/ConcurrentThetaContext.java

```diff
@@ -0,0 +1,58 @@
+package com.yahoo.sketches.theta;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * @author eshcar
+ */
+public class ConcurrentThetaContext extends UpdateSketchComposition {
```

Comment:
public ?

Reply:
Yes, this is the channel through which threads update the sketch - the application needs a context per thread.

Reply:
What would be really useful would be a simple example (could be a test) that would not need to process much data, but serves as an example of how to set-up and use the concurrent sketch in an actual multi-threaded environment, with clear comments that explain to the user what each part of the code is doing.